core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/OnBoardingViewModel.kt

```diff
@@ -53,14 +54,24 @@ class OnBoardingViewModel(
         }
     }
 
-    fun onClickNext(){
+    fun onClickExistJoining(){
         settings[PREF_TAG] =  true.toString()
         navController.navigate(RedirectViewModel.DEST_NAME, buildMap {
             putFromSavedStateIfPresent(ARG_NEXT)
             putFromSavedStateIfPresent(ARG_OPEN_LINK)
         })
     }
 
+
+    fun onClickIndividual() {
+        navController.navigate(IndividualLearnerViewModel.DEST_NAME, buildMap {
+        })
+    }
+    fun onClickAddNewOrganization() {
+        //Redirect to website (ask mike that is there prebuild util class to open website)
```

Comment:
This can be done using OpenExternalLinkUseCase . You can use the IDE to find usages of it. You can get it using the dependency injection.